.changeset/green-tools-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+"hardhat": patch
+---
+
+Clear account code on forking
```

Comment:
Fixed a problem related to configured accounts during forking. Some of them may be delegated in mainnet, making them unusable for local development. They are now undelegated after forking to turn them into normal EOAs.